rmgpy/reaction.py

```diff
@@ -85,6 +85,7 @@ class Reaction:
                                                     If ``False`` (by default), this library reaction will not be explored.
                                                     Only unimolecular library reactions with high pressure limit kinetics should be flagged (not if the kinetics were measured at some relatively low pressure)
     `comment`           ``str``                     A description of the reaction source (optional)
+    `is_forward`        ``bool``                    Indicates if the reaction was generated in the forward (true) or reverse (false)
```

Comment:
Could you merge this with the `isForward` attribute of TemplateReaction? Either by renaming this to `isForward` or by replacing all instances of `isForward` with `is_forward`.

Reply:
done